CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 
 - `alacritty migrate` failing with nonexistent imports
+- Associated text not sent for `Shift + <number>` when using kitty protocol
```

Comment:
Is it an accurate changelog entry? It seems to fix any `shift + alt + <char>` key combination? Just making sure that the changelog entry is as precise as possible 👍🏻

Reply:
Yeah, I'd need one more entry.